week3.Rmd

```diff
@@ -0,0 +1,147 @@
+date: "Last knitted `r format(Sys.Date(), '%d %b %Y')`"
+output:
+  pdf_document:
+    toc: yes
+    toc_depth: '3'
+  html_document:
+    df_print: paged
+    number_sections: yes
+    toc: yes
+    toc_float: yes
+    toc_depth: 3
+    code_folding: show
+
+editor_options:
+  chunk_output_type: inline
+---
+> Click into a code cell (the gray blocks below) to select or edit it. To run a selected code cell, hit `Ctrl`+`Shift`+`Enter` (Windows & Linux) / `Command`+`Shift`+`Enter` (MacOS). Make sure that each code cell runs successfully before you move on to the next one.
+
+Load the libraries that we'll use in this Problem Set: 
+```{r load libraries}
+library(tidyverse)
+library(dplyr)
+library(ggplot2)
+```
+
+##Introduction
+Ocean acidification is a hot topic when it comes to present day climate change and global warming. In this problem set, we'll explore the how and why behind ocean acidification by using chemical reactions to explain the process and apply pH and saturation state calculations through R coding methods.  
+
+##Learning Goals
+By the end of this exercise, you will be able to:
+1) Explain what ocean acidification is and how it works
+2) Derive the chemical reactions that relate to ocean acidification and carbonate dissolution
+3) Use the tibble function to make a data frame with various values related to ocean acidification in seawater
+4) Make calculations using your data frame values to compare the effects of ocean acidification on different seawater chemistries
+
+#Prerequisites
+Prior to completing this Problem Set, it is expected that students: 
+-Have a basic understanding of general chemistry, specifically with terms like acid, pH, and dissolution
+-Have a basic familiarity with the structure and syntax of code in R
+-Know how to copy, cut, comment-out, and edit text and code in R
+
+##Background and Defining Terms
+! [Alt text] (/Documents/CU Boulder Classes/Computational Tools/2020-ps-jolsenvaldez/Ocean-Acidification_image.png)
+**Ocean acidification** refers to the process by which *increasing levels of carbon dioxide* ($CO_2$) correlate to a *decrease in the pH of the ocean*. The earth has a natural process by which is sequesters carbon dioxide (in rocks, for example) and releases carbon dioxide (by trees, for example). However, anthropogenic ("human-caused") behavior has resulted in an unnatural increase in the amount of carbon dioxide in the atmosphere, which also leads to higher levels of $CO_2$ in the oceans. 
+
+But, what's "pH" again? Well, acidity is controlled by how many $H^+$ ions are in a solution (acids release $H^+$ ions) and the pH scale is used to describe the *concentration* of $H^+$ ions, which is represented by the term $[H^+]$. A solution is said to be acidic when the pH is below 7, and basic when the pH is above 7. pH and $[H^+]$ have a log relationship, meaning that a small change in pH can cause a big change in the $H^+$ concentration of a solution, like the ocean. 
+
```

Comment:
This is kind of picky, but where you say "meaning that a small change in pH can cause... " might be misleading to students, since pH is just a descriptive scale that we use - a change in pH doesn't cause a change in H+ concentration, it just describes a change that has occurred. Could be re-phrased something like, "Even a small change on the pH scale indicates a large change in H+ concentration, due to the log relationship between pH and H+."